src/components/application_manager/include/application_manager/commands/hmi/navi_set_video_config_request.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2017 Xevo Inc.
```

Comment:
Actually I talked with our legal department guys, and they requested me to include our company name for newly added files. I hope this is OK for you.